misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -156,8 +156,6 @@
     | zippy-materialized.* \| .* halting\ process:\ Server\ started\ with\ requested\ generation
     # Don't care for ssh problems
     | fatal:\ userauth_pubkey
-    # TODO(def-) Remove when #29199 is fixed
```

Comment:
We have to keep this ignore for 3 weeks so that old versions exit correctly too.

Reply:
Thanks, I added it back.